src/curl.c

```diff
@@ -95,7 +95,7 @@ int swupd_curl_init(void)
 		i = 1;
 
 		tok = strtok_r(str, ":", &ctx);
-		do {
+	        while (tok) {
```

Comment:
Please run `make compliant` to make the whitespace here consistent with the rest of the project.

Reply:
Done.